hooks/emitter/gov.go

```diff
@@ -198,6 +198,20 @@ func (h *Hook) handleMsgVote(
 	detail["title"] = proposal.Title
 }
 
+func (h *Hook) handleMsgCancelProposal(msg *v1.MsgCancelProposal) {
+	h.Write("REMOVE_DEPOSIT", common.JsDict{
+		"proposal_id": msg.ProposalId,
+	})
+
+	h.Write("REMOVE_VOTES", common.JsDict{
+		"proposal_id": msg.ProposalId,
+	})
+
+	h.Write("REMOVE_PROPOSAL", common.JsDict{
+		"id": msg.ProposalId,
+	})
+}
```

Comment:
I think we need to addAccountInTx/update the balance for all deposit addresses since their balances will also change.